includes/ocos.h

```diff
@@ -60,6 +60,18 @@ struct OrtTensorDimensions : std::vector<int64_t> {
   bool IsScalar() const{
     return empty();
   }
+
+  bool IsVector() const{
+    if (size() == 1) {
+      return true;
+    }
+
+    if ((size() == 2) && (*begin() == 1)) {
```

Comment:
Is a batch parameter not supported?

Reply:
Because we usually represent vector as [n] or [1,n]. It rarely meets [1,n,1] or [1, 1, n].
Currently, `IsVector` only supports `[n]` format.